ebs/ebs.go

```diff
@@ -179,10 +183,12 @@ func Init(root string, config map[string]string) (ConvoyDriver, error) {
 		if err := checkVolumeType(volumeType); err != nil {
 			return nil, err
 		}
+		kmsKeyId := config[EBS_DEFAULT_VOLUME_KEY]
```

Comment:
the kms key in the api call to aws is conditional on this value being non empty. the request here is to the internal service abstraction. a previous version of this pull request did only conditionally add the string to the volume info, but that was removed per previous review comments.